mysite/announcements/models.py

```diff
@@ -32,3 +31,24 @@ def clean(self):
             raise ValidationError("The date and time can't be in past")
         if self.date_time_expire <= self.date_time_to_publish:
             raise ValidationError("Scheduled time can't be less than expiry time")
+
+
+STATUS_CHOICES = (
```

Comment:
As you can see from documentation of [`choices`](https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.Field.choices), 

> Generally, it’s best to define choices inside a model class, and to define a suitably-named constant for each value:

So, we should also follow the same.